vector/src/main/java/im/vector/app/core/di/SingletonModule.kt

```diff
@@ -189,15 +189,22 @@ object VectorStaticModule {
         return GlobalScope
     }
 
-    @Provides
-    fun providesAnalyticsConfig(): AnalyticsConfig {
-        return analyticsConfig
-    }
-
     @Provides
     fun providesPhoneNumberUtil(): PhoneNumberUtil = PhoneNumberUtil.getInstance()
 
     @Provides
     @Singleton
-    fun providesBuildMeta() = BuildMeta()
+    fun providesBuildMeta() = BuildMeta(
+            isDebug = BuildConfig.DEBUG,
+            sdkInt = Build.VERSION.SDK_INT,
```

Comment:
`sdkInt` is a bit out of scope for `BuildMeta` object (I just see that `BuildMeta` is not new, was not aware of this class before), since `Build.VERSION.SDK_INT` have different values at runtime (and not at compile time). We have `BuildVersionSdkIntProvider` on the SDK, and also `AndroidVersionTestOverrider` (that looks a bit hacky, I would be happier to have a similar `BuildVersionSdkIntProvider`) app side, which allow to do some test and overriding.

Reply:
used the existing provider https://github.com/vector-im/element-android/pull/6517/commits/3bfe25a49f8ae397ae7d45d3ac10c8db230f3427